lib/puma/cluster.rb

```diff
@@ -81,7 +81,7 @@ def spawn_workers
           pid = spawn_worker(idx, master)
         end
 
-        debug "Spawned worker: #{pid}"
+        log "Spawned worker: #{pid}, phase: #{@phase}"
```

Comment:
There's already a log line on worker boot: https://github.com/chjasonwu/puma/blob/38724d4666aac0980ebd0779fd33ddef1c3ec468/lib/puma/cluster.rb#L477

Do we need one for spawn as well? If we are adding it, should the formatting match the boot log to include the indentation and index?
```suggestion
        log "- Worker #{idx} (PID: #{pid}) spawned, phase: #{@phase}"
```

<br>

IMO this does seem like more of a debug-only log, but that's for the maintainers to decide.